docker/iceberg-rest-adapter-image/README.md

```diff
@@ -0,0 +1,87 @@
+<!--
+  - Licensed to the Apache Software Foundation (ASF) under one
+  - or more contributor license agreements.  See the NOTICE file
+  - distributed with this work for additional information
+  - regarding copyright ownership.  The ASF licenses this file
+  - to you under the Apache License, Version 2.0 (the
+  - "License"); you may not use this file except in compliance
+  - with the License.  You may obtain a copy of the License at
+  -
+  -   http://www.apache.org/licenses/LICENSE-2.0
+  -
+  - Unless required by applicable law or agreed to in writing,
+  - software distributed under the License is distributed on an
+  - "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  - KIND, either express or implied.  See the License for the
+  - specific language governing permissions and limitations
+  - under the License.
+  -->
+
+# Iceberg rest adapter image
+
+For converting different catalog implementations into a rest one.
+
+## Building the Docker Image locally
+
+If you want to make changes to the local files, and test them out, you can build the image locally and use that instead:
+
+```bash
+# Navigate to the iceberg root directory
+cd ../../ 
+
+# Build the project
+./gradlew build -x test -x integrationTest
```

Comment:
Since it is a manual process, user can choose if they want to build only open api module. 

I prefer clean build as most of the things are needed for this module.